demo/src/app/dashboard/dashboard.component.ts

```diff
@@ -16,8 +16,8 @@ export class DashboardComponent implements OnInit {
 
   constructor(
     private heroService: HeroService,
-    @Inject('netlify.request') @Optional() request?: Request,
-    @Inject('netlify.context') @Optional() context?: Context,
+    @Inject(REQUEST) @Optional() request?: Request,
+    @Inject(REQUEST_CONTEXT) @Optional() context?: Context,
```

Comment:
I forgot to convert that in previous PR (`netlify.request` -> `REQUEST`) after setting up AppEngine, so this both moves that and use new place to get `REQUEST` and `REQUEST_CONTEXT` tokens from